libp2p/Session.h

```diff
@@ -110,14 +107,12 @@ class Session: public std::enable_shared_from_this<Session>
 	std::array<byte, 65536> m_data;			///< Buffer for ingress packet data.
 	bytes m_incoming;						///< Read buffer for ingress bytes.
 
-	PeerInfo m_info;						///< Dynamic information about this peer.
-
 	unsigned m_protocolVersion = 0;			///< The protocol version of the peer.
```

Comment:
What does this have to do with Host::protocolVersion()?
Also, this code isn't within scope of the PR.